app/_locales/en/messages.json

```diff
@@ -2539,12 +2539,12 @@
   "transactionHistoryBaseFee": {
     "message": "Base fee (GWEI)"
   },
-  "transactionHistoryEffectiveGasPrice": {
-    "message": "Effective gas price"
-  },
   "transactionHistoryPriorityFee": {
     "message": "Priority fee (GWEI)"
   },
+  "transactionHistoryTotalGasFee": {
+    "message": "Total Gas Fee"
```

Comment:
I like that every word starts with a capital letter. We should ideally do it for all labels in the Transaction Detail modal, but that's for another PR: 
![image](https://user-images.githubusercontent.com/80175477/128186075-b9bd79fd-4549-45a9-b703-6ee29b5695f8.png)